src/ServerWide/Operations/AddDatabaseNodeOperation.ts

```diff
@@ -45,7 +45,7 @@ class AddDatabaseNodeCommand extends RavenCommand<DatabasePutResult> implements
     createRequest(node: ServerNode): HttpRequestParameters {
         let uri = node.url + "/admin/databases/node?name=" + this._databaseName;
 
-        if (node) {
+        if (node && this._node) {
```

Comment:
why we need `if (node` check ?

Reply:
If node is null/undefined it will throw any way in line 46

Reply:
* fixed
* refactored` this._node` to `this._nodeTag` (I hope that is ok)